osd/aws/InstallFailed_ProxyBadTLS.json

```diff
@@ -0,0 +1,11 @@
+{
+  "severity": "Error",
+  "service_name": "SREManualAction",
+  "summary": "Installation blocked: Proxy malfunction",
+  "description": "Your cluster's installation is blocked because the configured cluster-wide proxy is not properly initiating TLS handshakes for HTTPS connections. Please review your network and proxy configuration and retry cluster installation. Please refer to the following documentation on cluster-wide proxies: https://docs.openshift.com/rosa/networking/configuring-cluster-wide-proxy.html#.",
```

Comment:
"Please" said in two consecutive sentences feels weird.

Reply:
Addressed in the latest commit